PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java

```diff
@@ -86,6 +89,27 @@ public void onCreate(Bundle savedInstanceState) {
         getmDbHandler().open();
         setContentView(R.layout.game_activity);
 
+        healing = findViewById(R.id.progress_healing);
+        health = findViewById(R.id.progress_health);
+        invisibility = findViewById(R.id.progress_invisibility);
+        telepathy = findViewById(R.id.progress_telepathy);
+
```

Comment:
I have not used butterknife library here and has went forward with existing code structure since my next task is to refactor the code according to changes in infra branch where butterknife is already used.